src/containers/DeepLink/LoginRequestComplete/LoginRequestComplete.js

```diff
@@ -66,8 +66,13 @@ const LoginRequestComplete = props => {
 
   const completeDeeplink = async () => {
     if (redirectinfo && redirectinfo.uri) {
-      await tryRedirect();
-      cancel();
+      const returnedData = await tryRedirect();
+      if (reDirect) {
+        // Only return to services, default is main home screen.
+        props.navigation.navigate("ServicesHome", { screen: reDirect, params: { data: returnedData.data }});
```

Comment:
I don't think this should be called redirect for a few reasons. Firstly, a redirect for a deeplink is already a defined term (redirect back to website), and secondly, it forces the user to navigate back to ServicesHome. I think you should change this to "fromService" or "sourceServiceId" or something along those lines to better indicate what it is.

Also, if this loginConsentRequest contains a redirect instead of a webhook, isn't this a no-op? Shouldn't the navigation happen first, then tryRedirect gets called?